### ISSUE TITLE
`ignore` without `select` seems to work in reverse

### ISSUE THREAD

**Issue by bersbersbers** [charliermarsh/ruff#3692]
I'll show here various numbers of errors reported by `ruff` based on `pyproject.toml` settings:

(Nothing): 12

`[tool.ruff]`: 0

`[tool.ruff]; select = ["ALL"]`:  359
`[tool.ruff]; select = ["ALL"]; ignore = ["D101", "D102", "D103", "ANN101", "RET504"]`:  109

`[tool.ruff]; ignore = ["D101", "D102", "D103", "ANN101", "RET504"]`:  250 (curiously, equals 359 - 109)

`[tool.ruff]; select = ["D101"]`:  7
`[tool.ruff]; ignore = ["D101"]`:  7

`[tool.ruff]; select = ["D102"]`:  95
`[tool.ruff]; ignore = ["D102"]`:  95

etc.

In summary, I think if nothing is `select`ed at all, `ignore` selects rather than ignores.

ruff 0.0.258

**Comment by charliermarsh** [comment]
Hey sorry — this should be fixed in 0.0.259 which we just released. There was a regression here in 0.0.258. Just LMK if you’re still seeing this after upgrading!

**Comment by bersbersbers** [comment]
All good now, thanks :)
